java-monitoring/run_protoc.sh

```diff
@@ -0,0 +1,360 @@
+        mv "${LIBRARY_GEN_OUT}"/google/monitoring/v3/gapic-google-cloud-monitoring-v3-java-srcs_pkg/gapic-google-cloud-monitoring-v3-java/src/main/java/samples "${LIBRARY_GEN_OUT}"/google/monitoring/v3/gapic-google-cloud-monitoring-v3-java-srcs_pkg/gapic-google-cloud-monitoring-v3-java
+    fi
+done
+
+for proto_src in ; do
+    mkdir -p "${LIBRARY_GEN_OUT}"/google/monitoring/v3/gapic-google-cloud-monitoring-v3-java-srcs_pkg/gapic-google-cloud-monitoring-v3-java/src/main/proto
+    cp -f --parents "${proto_src}" "${LIBRARY_GEN_OUT}"/google/monitoring/v3/gapic-google-cloud-monitoring-v3-java-srcs_pkg/gapic-google-cloud-monitoring-v3-java/src/main/proto
+done
+
+for test_src in ${LIBRARY_GEN_OUT}/google/monitoring/v3/monitoring_java_gapic_srcjar-test.srcjar; do
+    mkdir -p "${LIBRARY_GEN_OUT}"/google/monitoring/v3/gapic-google-cloud-monitoring-v3-java-srcs_pkg/gapic-google-cloud-monitoring-v3-java/src/test/java
+    unzip -q -o "${test_src}" -d "${LIBRARY_GEN_OUT}"/google/monitoring/v3/gapic-google-cloud-monitoring-v3-java-srcs_pkg/gapic-google-cloud-monitoring-v3-java/src/test/java
+    rm -r -f "${LIBRARY_GEN_OUT}"/google/monitoring/v3/gapic-google-cloud-monitoring-v3-java-srcs_pkg/gapic-google-cloud-monitoring-v3-java/src/test/java/META-INF
+done
+cd "${LIBRARY_GEN_OUT}"/google/monitoring/v3/gapic-google-cloud-monitoring-v3-java-srcs_pkg/gapic-google-cloud-monitoring-v3-java/..
+tar -zchpf gapic-google-cloud-monitoring-v3-java/gapic-google-cloud-monitoring-v3-java.tar.gz gapic-google-cloud-monitoring-v3-java/*
+cd - > /dev/null
+mv "${LIBRARY_GEN_OUT}"/google/monitoring/v3/gapic-google-cloud-monitoring-v3-java-srcs_pkg/gapic-google-cloud-monitoring-v3-java/gapic-google-cloud-monitoring-v3-java.tar.gz "${LIBRARY_GEN_OUT}"/google/monitoring/v3/gapic-google-cloud-monitoring-v3-java-srcs_pkg.tar.gz
+
+##################### Section 3 #####################
+# generate proto-google-cloud-monitoring-v3-java-resources.tar.gz
+#####################################################
+cd "${GOOGLEAPIS_ROOT}"
+"${PROTOC_ROOT}"/protoc "--java_out=${LIBRARY_GEN_OUT}/google/monitoring/v3/monitoring_proto-speed-src.jar" \
+google/monitoring/v3/alert.proto \
+google/monitoring/v3/alert_service.proto \
+google/monitoring/v3/common.proto \
+google/monitoring/v3/dropped_labels.proto \
+google/monitoring/v3/group.proto \
+google/monitoring/v3/group_service.proto \
+google/monitoring/v3/metric.proto \
+google/monitoring/v3/metric_service.proto \
+google/monitoring/v3/mutation_record.proto \
+google/monitoring/v3/notification.proto \
+google/monitoring/v3/notification_service.proto \
+google/monitoring/v3/query_service.proto \
+google/monitoring/v3/service.proto \
+google/monitoring/v3/service_service.proto \
+google/monitoring/v3/snooze.proto \
+google/monitoring/v3/snooze_service.proto \
+google/monitoring/v3/span_context.proto \
+google/monitoring/v3/uptime.proto \
+google/monitoring/v3/uptime_service.proto
```

Comment:
It looks like a repeat of the list of protos. Can we shove it into a variable?

Reply:
done